File: catchCat/app/src/main/java/com/lq/catchcatbasicgame/MainActivity.java
package com.lq.catchcatbasicgame;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {
      Intent intent;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void start(View view)
    {
         intent = new Intent(MainActivity.this, MainActivity2.class);
         startActivity(intent);
    }

}

File: RunCatGame/app/src/main/java/com/lq/runcatgame/MainActivity.kt
package com.lq.runcatgame

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AlertDialog
import com.lq.runcatgame.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
     lateinit var  binding : ActivityMainBinding;
     val shared : DataStore = DataStore.getInitStore();
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater);
        setContentView(binding.root)
        binding.startButton.setOnClickListener{
            val intent = Intent(this@MainActivity,GameActivity::class.java);
            this@MainActivity.startActivity(intent);
            finish();
        }
        binding.scoreButton.setOnClickListener(object: View.OnClickListener{
            override fun onClick(p0: View?) {
                  AlertDialog.Builder(this@MainActivity)
                      .setTitle("HIGH SCORE")
                      .setMessage("High Score : "+shared.getHightScore(this@MainActivity))
                      .setNegativeButton("OK",{dialog,i -> dialog.cancel()}).show();
            }


        })
    }



}